src/libraries/Common/src/Internal/Cryptography/ILiteSymmetricCipher.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Security.Cryptography;
+
+namespace Internal.Cryptography
+{
+    internal interface ILiteSymmetricCipher : IDisposable
```

Comment:
Do we need an interface here?  Any reason it can't be done on BasicSymmetricCipher?

Reply:
> Any reason it can't be done on BasicSymmetricCipher?

`BasicSymmetricCipher` has a property for the IV which is one of the things I was trying to avoid. We could just have a null IV, but I was trying to be super careful about the readability of this. One could assume "no IV means it is ECB" when really what we were intending was a CBC with no captured IV since it is not reusable.

The exception to this is the BCrypt implementation. It has no "Lite" implementation because we are responsible for passing in the IV for each call. So the BCrypt implementation just implements the interface without extracting the functionality.